### ISSUE TITLE
Detroit - Missing Translations 

### ISSUE THREAD

**Issue by kathyccheng** [CityOfDetroit/bloom#1093]
https://docs.google.com/spreadsheets/d/1KmIs93OKQIDsKyay90MLWmItLVBCyAytJJ8s5p-_Zm8/edit#gid=290561964

Jared will take the resources ones highlighted in purpe

**Comment by emilyjablonski** [comment]
Strings with issues:

- Translating the accessibility features
These strings are not stored on the listing but in translation files, so we would need explicit translations for these. I can just google translate them and add them in, if we were going to google translate anyway?

- Translating program titles on the tags on the directory page
We don’t have google translate set up on the directory page, so while these program tags next to the images translate on the details page, they aren’t translating on the directory page, so we probably need to do work to setup google translate on the directory page to make that happen

- Translating program titles in the filters modal (similar to above, sort of)
We aren’t using translation strings for program titles, since we decided to google translate the program strings from what is stored in the db. In the filter modal we fetch the filters, and display the titles, which are stored in English. I’m not sure how to (a) pass this to google translate since it’s on the directory page and isn’t based on one listing or (b) use the keys we did get translated since there is not a consistent key on programs to tie the strings to (I don’t think we can use the titles since they have spaces?).

- Row 8 (“For listings questions…“), Row 44 (“For each type of unit…“), Row 46 (“We know that finding…“), Row 47 (“The CDHRD would like to thank…“)
We are concatenating strings to build this, so we would need the strings we are concatenating translated separately or to remove the styling. The solutions to prevent this problem are to (1) not concatenate strings where possible and (2) where we have to, ensure that the keys we are getting translated are the strings from the translation file, not taken directly from the site
